### ISSUE TITLE
Don't emit tensor values in benchmark JSON

### ISSUE THREAD

**Issue by awf** [microsoft/knossos-ksc#1028]
Currently the pytest-benchmark JSON outputs are emitting the precise tensor values on which benchmarks are run.  
This will lead to large files when we use larger tensors (e.g. vsqrl).  
![image](https://user-images.githubusercontent.com/128119/130616752-93f51c6c-e778-4852-82fc-aad738d34f84.png)

Proposal: don't emit the config parameters

Pro: smaller benchmark JSON files

Con: Can't rerun benchmark on same values.  I feel this is a minimal con -- benchmark timings are impossible to reproduce exactly anyway, and bugs triggered by unique values are rare enough that a huge long-term file size feels a poor tradeoff.  
If we want to acheive that, we can make the _config functions more deterministic.

**Comment by awf** [comment]
@cgravill, thoughts?

**Comment by cgravill** [comment]
Yes, partial values don't help much anyway so skipping might be easier. I'd have to investigate to see how to avoid outputting though.

Outputting the seed might help with reproducibility (if we were to properly pin versions etc.) but is separate.
